core/tests/org.fusesource.ide.camel.tests.util/src/org/fusesource/ide/camel/tests/util/CommonTestUtils.java

```diff
@@ -103,14 +116,17 @@ public static void createScreenshotFolder(String folder) {
 	public static void enablePerspectiveSwitchPreset() {
 		IPreferenceStore store = IDEWorkbenchPlugin.getDefault().getPreferenceStore();
 		store.setValue(IDEInternalPreferences.PROJECT_SWITCH_PERSP_MODE, IDEInternalPreferences.PSPM_ALWAYS);
-		
+		DebugUIPlugin.getDefault().getPreferenceStore().setValue(IInternalDebugUIConstants.PREF_SWITCH_PERSPECTIVE_ON_SUSPEND, IDEInternalPreferences.PSPM_ALWAYS);
 	}
 	
 	/**
 	 * closes all editors
 	 */
 	public static void closeAllEditors() {
-		PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage().closeAllEditors(false);
+		IWorkbenchPage page = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage();
+		if (page != null) {
```

Comment:
I don't think it can occur in this context.
The active page can be null when there was no editor that was able to open but if the workbench window does not open I think the test is not even launched.

Reply:
Will add the check anyway, maybe there are some other cases